x-pack/plugins/observability_solution/observability/server/services/slo/transform_generators/synthetics_availability.ts

```diff
@@ -77,7 +77,7 @@ export class SyntheticsAvailabilityTransformGenerator extends TransformGenerator
        * to build a URL back to Synthetics */
       ...(includesDefaultGroupings && {
         'observer.name': { terms: { field: 'observer.name' } },
-        config_id: { terms: { field: 'config_id' } },
+        'monitor.config_id': { terms: { field: 'config_id' } },
```

Comment:
@shahzad31 
Even though `config_id` is at the root in Synthetics, I want to namespace the value in the SLO documents. For this reason I've decided to add it to the `monitor` namespace. Is that okay with you?

Reply:
++